### ISSUE TITLE
Add more translations

### ISSUE THREAD

**Issue by humdingerb** [HaikuArchives/KeymapSwitcher#6]
I added the catkeys to [Polyglot](https://i18n.kacperkasper.pl).
People can add/update translations there. Devs preparing a new release of KeymapSwitcher should export all catkeys from there first.

**Comment by TejasMate** [comment]
Here, I translated Catkeys for [Hindi](https://i18n.kacperkasper.pl/files/24/lang/26) Language

**Comment by diversys** [comment]
Is there a way to export all languages as zip file or something? How do one know if some strings changed in already translated languages?

**Comment by TejasMate** [comment]
No, from [polyglot](https://i18n.kacperkasper.pl/projects) only .catkeys files are exported and I think, there is no such feature whether the strings have been changed from the already translated language. Admin @humdingerb should know about this.

**Comment by humdingerb** [comment]
Yes. Below all translations of a project is an "Export all" link that downloads all catkeys in a zip file.
Before the project releases a new version, I'd recommend downloading and unpacking that zip in the "locales" folder of the project to have everthing up-to-date.

**Comment by diversys** [comment]
I don't seem to have "Export all" link.

![image](https://user-images.githubusercontent.com/1316513/73089731-eb0bad80-3ee7-11ea-8a76-7f350f642dad.png)

**Comment by humdingerb** [comment]
Oh. It might be that only owner/admin of the project sees that.
I opened a ticket to ask KapiX if we could have the link visible for everyone:
https://github.com/KapiX/Polyglot/issues/53

**Comment by humdingerb** [comment]
@diversys, KapiX has fixed the issue. You should be able to see and use the "Export all" now.

**Comment by diversys** [comment]
Thanks, it works now, I'll try to find some time to update the catkeys.

**Comment by humdingerb** [comment]
I created a PR: #17
